### DISCUSSION TITLE
Refrigerator Energy Monitoring

### DISCUSSION THREAD

**Discussion by PlatGitHub468** [nVuln/homebridge-lg-thinq#49] [Ideas]
Hi,

In the ThinQ app for my refrigerator, there is an option called "Energy Monitoring" where the app displays the daily, weekly, monthly usage  (kWh).  Is it possible to get this data expose through Homebridge ?

**Comment by nVuln** [comment]
do homekit support that type?
I just researched, official Homekit do not support, only 3rd Eve home app

**Reply by PlatGitHub468** [reply]
I don't think HomeKit support it but I am also using third party Eve App since I can see power/energy options for some of the devices (Wemo Insight Switch, Kasa HS110...).  It would be nice to see the LG refrigerator power consumption as well in the Eve App.
